spec/lib/blacklight/solr/search_builder_spec.rb

```diff
@@ -553,7 +553,71 @@
       end
     end
   end
-  
+
+  describe "#add_facet_paging_to_solr" do
+    let(:facet_field) { 'format' }
+    let(:sort_key) { Blacklight::Solr::FacetPaginator.request_keys[:sort] }
+    let(:page_key) { Blacklight::Solr::FacetPaginator.request_keys[:page] }
+
+    let(:blacklight_config) do
+      Blacklight::Configuration.new do |config|
+        config.add_facet_fields_to_solr_request!
+        config.add_facet_field 'format'
+        config.add_facet_field 'format_ordered', :sort => :count
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.